### PR TITLE
Update TfL Image Processing tutorial

### PR DESCRIPTION
**Pull request by SteveRosam**

### CODE REVIEW

File: docs/platform/tutorials/image-processing/index.md [hunk 1]
```diff
@@ -66,7 +66,11 @@ There are four stages to the processing pipeline you build in this tutorial:
 
     - Detect objects within images
 
-4. Web UI configuration
+4. Stream merge
+
+    - Merge the seperate data streams into one
```
Comment (tbedford) [inline docs/platform/tutorials/image-processing/index.md]: Typo. Separate.

**Review by tbedford — CHANGES_REQUESTED** [review]
Typo, plus the numbering seems to have gone slightly wrong:

image

This is also true for the **stream merge** topic, which is 6 in TOC but showing as 5 in topic.

File: docs/platform/tutorials/image-processing/index.md (hunk 1 above)
Comment (SteveRosam) [reply docs/platform/tutorials/image-processing/index.md]: done and done

**Comment by SteveRosam** [comment]
@tbedford this is ready for round 2

**Review by tbedford — APPROVED** [review]
LGTM

**Pull request merged**